.storybook/webpack.config.js

```diff
@@ -89,15 +90,24 @@ module.exports = ({ config }) => {
   config.module.rules.push({
     test: /\.scss$/,
     use: [
-      {
-        loader: devMode ? "style-loader" : MiniCssExtractPlugin.loader,
-      },
+      // {
+      //   loader: MiniCssExtractPlugin.loader,
+      // },
       {
         loader: 'css-loader',
         options: {
           importLoaders: 1,
           modules: {
-            localIdentName: '[local]-[hash:base64:5]',
+            localIdentName: '[hash:base64]',
+            localIdentHashFunction: 'md4',
+            localIdentContext: path.resolve(__dirname, "../packages/bpk-stylesheet"),
+            mode: (resourcePath, resourceQuery, resourceFragment) => {
+              if (/global.scss$/i.test(resourcePath)) {
+                return "global";
+              }
+
+              return "local";
+            },
```

Comment:
This is setup specifically to generate the classes the exact same way that `bpk-stylesheet` does, it relies on the hashing function being totally deterministic for both of the separate `build`s to lineup. 

I don't think we need the `mode` function here, that is just left over from experimentation.